File: AkkaWithZeroMQ/src/main/java/org/akka/essentials/zeromq/example4/MyActorSystem.java
package org.akka.essentials.zeromq.example4;

import akka.actor.ActorSystem;
import akka.actor.Props;

public class MyActorSystem {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		ActorSystem system = ActorSystem.create("zeromqTest");
		system.actorOf(new Props(PushActor.class), "push");
		system.actorOf(new Props(PullActor1.class), "pull1");
		system.actorOf(new Props(PullActor2.class), "pull2");
	}

}

File: AkkaWithZeroMQ/src/main/scala/org/akka/essentials/zeromq/example4/MyActorSystem.scala
package org.akka.essentials.zeromq.example4

import akka.actor.ActorSystem
import akka.actor.Props

object MyActorSystem {

  def main(args: Array[String]): Unit = {
    val system = ActorSystem("zeromqTest")
    system.actorOf(Props[PushActor], name = "push")
    system.actorOf(Props[PullActor1], name = "pull1")
    system.actorOf(Props[PullActor2], name = "pull2")
  }
}